tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -301,6 +306,31 @@ def _set_affinity(self, affinity_map_path):
         for layer in custom_affinity:
             if layer not in automatic_affinity:
                 raise ConfigError('Layer \'{layer}\' is not present in network'.format(layer=layer))
+        if isinstance(ng, UnsupportedPackage):
+            if not hasattr(self.network, 'layers'):
+                ng.raise_error('affinity setting')
+            self._set_affinity_via_layers(custom_affinity, automatic_affinity)
+            return
+        self._set_affinity_ng(custom_affinity, automatic_affinity)
```

Comment:
Could you invert the logic here to check for `.layers` first, and to only use ngraph if it's not available? That would make the code compatible with both the last release of OpenVINO and the most recent development build (which doesn't have `.layers` anymore). Then you could merge this PR while CI still uses the release build of OpenVINO.